.pre-commit-config.yaml

```diff
@@ -0,0 +1,39 @@
+---
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+      - id: black
+        args: [--skip-string-normalization]
```

Comment:
Why not just use the defaults?
```suggestion
```

Reply:
Fixed as you suggested.

I have been using single-quote for my python codes. But Python Black's standard seems to be double-quotes. I will switch to double-quotes. Will do this for Caper too later.